tests/resources/Selectors.robot

```diff
@@ -1,10 +1,10 @@
 *** Variables ***
-${XP LAUNCH SECTION}    xpath://h2[contains(@class, 'jp-Launcher-sectionTitle')][contains(text(), 'Starters')]
+${XP LAUNCH SECTION}    xpath://h2[contains(@class, 'jp-Launcher-sectionTitle')][text() = 'Starters']
 ${CSS LAUNCH CARD}    css:[data-category\="Starters"]
-${CSS LAUNCH CARD TUTORIAL}    ${CSS LAUNCH CARD}\[title\="A guided tutorial through using robotkernel"]
+${CSS LAUNCH CARD TUTORIAL}    ${CSS LAUNCH CARD}\[title\="Tutorial for Robot Framework on Jupyter"]
 ${XP FILE TREE ITEM}    xpath://span[contains(@class, 'jp-DirListing-itemText')]
-${XP FILE TREE TUTORIAL}    ${XP FILE TREE ITEM}\[contains(text(), 'robotkernel-tutorial')]
-${XP FILE TREE TUTORIAL 00}    ${XP FILE TREE ITEM}\[contains(text(), '00 JupyterLab User Interface.ipynb')]
+${XP FILE TREE TUTORIAL}    ${XP FILE TREE ITEM}\[text() = 'tutorial']
```

Comment:
this was the only thing you had missed for the most recent ci fail... but you know me, can't leave well enough alone...